Airports/EGGD/SMR/Geo.txt

```diff
@@ -121,6 +121,8 @@ N051.23.07.342 W002.42.20.087 N051.23.07.055 W002.42.20.064 smrYellow
 N051.23.06.876 W002.42.12.544 N051.23.08.181 W002.42.12.674 smrYellow
 ; Stand 19 Crossbar
 N051.23.08.172 W002.42.12.839 N051.23.08.183 W002.42.12.495 smrYellow
+; - Stand_18
```

Comment:
Would it be possible to have these in order, and re-number the comments for 13-15 (now 14-16) so people don't get confused in future? Also it seems you have added a Stand 18 Crossbar, but it's at the bottom of the file? Could you move that up to the relevant position and name it appropriately please.